homeassistant/components/input_boolean.py

```diff
@@ -47,18 +53,14 @@ def turn_off(hass, entity_id):
 
 def setup(hass, config):
     """Set up input boolean."""
-    if not isinstance(config.get(DOMAIN), dict):
-        _LOGGER.error('Expected %s config to be a dictionary', DOMAIN)
-        return False
-
     component = EntityComponent(_LOGGER, DOMAIN, hass)
 
     entities = []
 
-    for object_id, cfg in config[DOMAIN].items():
-        if object_id != slugify(object_id):
-            _LOGGER.warning("Found invalid key for boolean input: %s. "
-                            "Use %s instead", object_id, slugify(object_id))
+    for object_id, cfg, duplicate in dict_items_from_list(config[DOMAIN]):
```

Comment:
Oh, I get it. You have added a `PLATFORM_SCHEMA` instead of `CONFIG_SCHEMA`, so bootstrap is collecting your config. Change that and you can use the original `for object_id, cfg in config[DOMAIN].items():`